Sources/Client/Model/Channel/Channel+UnreadCount.swift

```diff
@@ -14,7 +14,7 @@ extension Channel {
     ///   - event: an event.
     func updateUnreadCount(event: Event) {
         switch event {
-        case .messageNew(let message, _, _, _):
+        case .messageNew(let message, _, _, _), .notificationMessageNew(let message, _, _, _, _):
```

Comment:
`.notificationMessageNew` no needed here. It's parsed in `Client+UnreadCount`.

Reply:
I've moved it here, since it was redundant.

Reply:
why?
`updateUnreadCount` won't be called for `. notificationMessageNew`

Reply:
Ok, I got it.